<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit_wind</class>
 <widget class="QDialog" name="credit_wind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Сумма кредита, руб.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_summ">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_term">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_term">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Срок кредита, мес.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_proc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Процентная ставка годовая, %</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ann">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_dif">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Дифференцированый</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Общая выплата, руб.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Макс. ежемесячный платеж, руб.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Переплата, руб.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_month">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_over">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_all">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>231</y>
     <width>220</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить расчет</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
